### PR TITLE
Refactor code and import luxon by npm

### PR DESCRIPTION
**Pull request by heintayzar-hm**
Refactor the code and add luxon lib for datetime

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,79 +1,28 @@
...
-load();
+load(books);
 
 // eventlistener on hash change
 window.addEventListener('hashchange', () => {
-  load();
+  load(books);
 });
 
 // intial color for links
```
Comment (omar-muhamad) [inline src/index.js:27]: - [ ] Clicking on the `add new` navigation option suppose to show a form to add a new book, but that's not happening. So please fix that.

chrome-capture-2022-9-31

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @heintayzar-hm,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.js (hunk 1 above)
Comment (heintayzar-hm) [reply src/index.js:27]: @Omar-Muhamad  I think you used something like live version or raw cdn run in local

Comment (omar-muhamad) [reply src/index.js:27]: I tried to add a new task and there is no form to add it so please fix your code so the app can function as expected.

Comment (heintayzar-hm) [reply src/index.js:27]: Are you running on http server? It is fine here?

Comment (heintayzar-hm) [reply src/index.js:27]: @Omar-Muhamad  Are you sure you are running on HTTP server? and also install npm module Here it is fine.
h

**Review by Lameck1 — COMMENTED** [review]
Hello @heintayzar-hm 👋,

_This is an additional review as a result of the second opinion request._

From my testing, the app is working as expected and the bug noticed by @Omar-Muhamad cannot be realized on my side.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please note that since we will approve it manually, this could take a few hours!

Keep rocking 🚀!

**Comment by omar-muhamad** [comment]
> Hello @heintayzar-hm 👋,
> 
> _This is an additional review as a result of the second opinion request._
> 
> From my testing, the app is working as expected and the bug noticed by @Omar-Muhamad cannot be realized on my side.
> 
> As this was the only reason that your project was not approved it will be approved as soon as possible. Please note that since we will approve it manually, this could take a few hours!
> 
> Keep rocking 🚀!

Hi @Lameck1,
I installed the project dependencies (libraries) and run the live server and nothing appeared on my device as shown in the GIF image above.
What Am I supposed to do in this case?

**Review by TadesseDev — APPROVED** [review]
### Hi @heintayzar-hm 
### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
ZLnP

## Highlights
 - [ ]  Good job overall following the template design 💯
 - [ ]  Correct GitHub Flow 👍🏾
 - [ ]  Good commit history 🥇
 - [ ]  All linters passed 🥇
## Optional suggestions

- [Optional] I have attached a video explaining the issue. I hope this helps. 👍

https://user-images.githubusercontent.com/69077061/199037304-69b0445b-cadc-4788-b956-c24c771fa993.mp4


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by heintayzar-hm** [comment]
> ### Hi @heintayzar-hm
> ### I'm Tadesse Alemayehu your reviewer for this PR. 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! tada ZLnP [ ZLnP ](https://user-images.githubusercontent.com/69077061/199034700-9c6acf99-6b08-46bb-9a0f-9dbbc97e60da.gif) [ ](https://user-images.githubusercontent.com/69077061/199034700-9c6acf99-6b08-46bb-9a0f-9dbbc97e60da.gif)
> 
> ## Highlights
> * [ ]   Good job overall following the template design 100
> * [ ]   Correct GitHub Flow 👍🏾
> * [ ]   Good commit history 1st_place_medal
> * [ ]   All linters passed 1st_place_medal
> 
> ## Optional suggestions
> * [Optional] I have attached a video explaining the issue. I hope this helps. +1
> 
>  2022-10-31.14-50-05.mp4 
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

@Tadesse-Alemayehu Thanks for sharing but actually it may not be the previous code reviewer error. Because that issue solved if we used HTTP server. (We cannot use modules in `file:// protocol`). As per the previous reviewer said, he run in the live server so the problem does not lie here. Anyways thank for sharing

**Comment by TadesseDev** [comment]
Sure @heintayzar-hm, As I have mentioned there are a lot of reasons for this to happen And many ways to around it. I just want to mention my findings.

**Pull request merged**